### DISCUSSION TITLE
Honeywell device Laser Scanner

### DISCUSSION THREAD

**Discussion by omkar243** [journeyapps/zxing-android-embedded#658] [Q&A] [answered]
Hello, can anyone please tell me this library also work with Honeywell device using Laser scanner which not having Camera feature? 
If yes, then share some code snippet to work with.

**Comment by rkistner** [comment, marked as answer]
This library is only designed to scan barcodes from camera images. To use an embedded laser scanner, you'd have to use Honeywell-specific APIs. I'm not sure about Honeywell specifically, but they typically have options to either listen for Intent broadcasts, or an SDK that you can use.

**Reply by omkar243** [reply]
okay.
